install/install.bat

```diff
@@ -0,0 +1,13 @@
+echo OFF
+echo okfn/messytables@%1
+echo scraperwiki/xypath@%2
+echo scraperwiki/databaker@%3
+ 
+pause
```

Comment:
Is this to check they should go ahead? Should probably have a prompt message before.